src/connect/components/tree-view-item/tree-view-item.tsx

```diff
@@ -32,16 +34,17 @@ export enum ItemStatus {
     Offline = 'offline',
 }
 
-export interface TreeItem<T extends string = string> {
+export interface TreeItem {
     id: string;
     label: string;
     type: ItemType;
+    path: string;
```

Comment:
Added path property to `TreeItem`.
IMPORTANT: `path` should include the drive: `my-drive-name-or-drive-id/path/to/the/item`